src/middlewares/error_handling.js

```diff
@@ -16,15 +16,16 @@ import {
 
 export default (logger) => (err, req, res, next) => {
   let status;
-
-  if (err instanceof ValidationError) {
+  if (err instanceof ValidationError || err.type === 'entity.parse.failed') {
     status = 400;
   } else if (err instanceof AuthenticationError) {
     status = 401;
   } else if (err instanceof PermissionError) {
     status = 403;
   } else if (err instanceof NotFoundError) {
     status = 404;
+  } else if (err.type === 'entity.too.large') {
+    status = 413;
```

Comment:
Do we need to change the status code here? Body parser seems to return 413 when the payload is too big: https://github.com/expressjs/body-parser/blob/e6ccf98015fece0851c0c673fc2776c30ad79e5d/test/json.js#L113-L121

We could add a test case to try this.

Reply:
It looks like this middleware catches this error and as the type is unknown, the 500 is thrown

Reply:
OK :+1: